CONTRIBUTING.md

```diff
@@ -49,15 +49,15 @@ To get started, run `bin/setup`
 
 After setting up, you can run the application using [foreman]:
 
-    % foreman start
+    foreman start
```

Comment:
@jessieay and at @dijonkitchen:  I've seen this come up a few times in some other conversations and the doc working group and it seems that the general consensus is to not have the prompt as it makes it easier to copy the command to run it.  Personally I like having it as it's a clear call out that it's a shell command, but what we could do instead is have it be wrapped in the code/monospace formatting (by using the back ticks) and indented (which I think it already is), like so:

    `foreman start`

And using slightly clearer language above, e.g., "After setting up, you can run the application by running the following [foreman] command in your shell:" or something to that effect.